server/nuxeo-nxr-server/src/main/resources/templates/common-base/client/scripts/taskGC.groovy

```diff
@@ -0,0 +1,33 @@
+import org.apache.logging.log4j.LogManager
```

Comment:
Could you rename the script to match the already existing one (consistency)? Such as `deleteOrphanTasks.groovy`.